flask/src/server.py

```diff
@@ -0,0 +1,39 @@
+import ffmpeg
+import tempfile
+import os
+
+from girder_client import GirderClient
+from flask import Flask, request, Response, send_file
+from flask_cors import CORS
+
+app = Flask(__name__)
+
+cors_domain = os.environ.get('CORS_DOMAIN', '*')
+girder_url = os.environ.get('GIRDER_API_URL', 'https://data.kitware.com/api/v1/')
+
+CORS(app, resources={r'/*': {'origins': cors_route}})
```

Comment:
I get the following when I test this:

```
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/app/server.py", line 14, in <module>
    CORS(app, resources={r'/*': {'origins': cors_route}})
NameError: name 'cors_route' is not defined
```

```cors_route``` needs to be ```cors_domain```

Reply:
This has been fixed, testing was successful.